packages/stock-location/src/utils/db-aware-column.ts

```diff
@@ -0,0 +1,26 @@
+import { Column, ColumnOptions, ColumnType } from "typeorm"
```

Comment:
todo: we don't support SQlite anymore so I think we can compately remove this file and usage